azure-key-vault-properties-provider/2.0/modules/ROOT/pages/azure-key-vault-properties-provider-reference.adoc

```diff
@@ -0,0 +1,48 @@
+= Azure Key Vault Properties Provider 2.0 Reference - Mule 4
+
+Azure Key Vault Properties Provider is a configuration that enables you to retrieve your secrets from a secure Azure cloud service before your Mule app runs.
+
+[[Config]]
+== Configuration
+
+Default Configuration for Azure Key Vault Properties Provider.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | Name for this configuration. Connectors reference the configuration with this name. | | x
+| Client Id a| String |  Client ID of your app. |  | x
+| Tenant Id a| String |  Tenant ID of your app. |  | x
+| Client Secret a| String |  Client secret of your app. |  | x
+| Key Vault Name a| String |  Name of the vault that you want to connect. |  | x
+| Uri Override a| String |  The URI override for the key vault. |  | 
```

Comment:
It seems like this could benefit from the fuller description given in your other PR. It's probably the same functionality?